web/src/lib/search/searchQuickActions.ts

```diff
@@ -0,0 +1,45 @@
+import { findMatchInBlock, findMatchInLabelledText } from "@/lib/search/helpers";
+import { Match } from "@/lib/search/typesForSearch";
+import { QuickAction } from "@/lib/types/types";
+
+export const searchQuickActions = (quickActions: QuickAction[], term: string): Match[] => {
+  const matches: Match[] = [];
+
+  for (const quickAction of quickActions) {
+    let match: Match = {
+      filename: quickAction.id,
+      snippets: [],
+    };
+
+    const content = quickAction.content.toLowerCase();
+    const name = quickAction.name.toLowerCase();
+    const cta = quickAction.ctaText?.toLowerCase();
+    const ctaLink = quickAction.ctaLink?.toLowerCase();
+    const filename = quickAction.filename?.toLowerCase();
+    const urlSlug = quickAction.urlSlug.toLowerCase();
+    const id  = quickAction.id.toLowerCase();
+    const form = quickAction.form?.toLowerCase();
+
+
+    const blockTexts = [content];
+    const labelledTexts = [
+      { content: cta, label: "CTA Text" },
+      { content: ctaLink, label: "CTA Link" },
+      { content: name, label: "Name/Title" },
+      { content: content, label: "Content" },
```

Comment:
[pebble] This is already in `blockTexts` and if we include it here as well then it would show up twice in the search tool

Reply:
Done